index.html

```diff
@@ -5569,13 +5569,8 @@ <h3 id='fields'>Fields</h3>
 <td><a href="https://www.hl7.org/fhir/datatypes.html#Identifier">Identifier</a></td>
 </tr>
 <tr>
-<td>name</td>
-<td>The name associated with the Claim.</td>
-<td><a href="https://www.hl7.org/fhir/datatypes.html#string">string</a></td>
-</tr>
-<tr>
 <td>status</td>
-<td>Indicates whether the Bill or Quote is presently active or cancelled. If a Bill is deleted or voided it will have a status of Cancelled. If a closed bill is corrected, the original bill will be voided with a status of Cancelled and a new bill will be created to replace it with an Active status.</td>
+<td>Indicates whether the Bill is presently active or cancelled. If a Bill is deleted or voided it will have a status of Cancelled. If a closed bill is corrected, the original bill will be voided with a status of Cancelled and a new bill will be created to replace it with an Active status.</td>
```

Comment:
I'll nitpick.
Why is bill capitalized in the first two sentences and not afterward? Seems we want to highlight it as like a reserved word, and it should be consistent probably.

Reply:
I dunno.  It was copied from Select's api docs this way orignally?
I'll fix it though.